src/index.js

```diff
@@ -258,7 +258,7 @@ walletRouter.get(
   const wallet = req.wallet;
   // Expects token uid
   const token = req.query.token || null;
-  const balance = wallet.getBalance(token);
+  const balance = wallet._getBalanceRaw(token);
```

Comment:
It seems dangerous to use internal methods that we can freely change without considering compatibility. Besides that, I feel that not all implementation of the facade have this method available.